src/sercon.c

```diff
@@ -336,8 +342,15 @@ static void sercon_1000(struct bregs *regs)
     }
 
     SET_LOW(sercon_enable, mode <= 0x07);
+    SET_LOW(sercon_col_last, 0);
+    SET_LOW(sercon_row_last, 0);
+    SET_LOW(sercon_attr_last, 0);
 
+    sercon_term_reset();
     sercon_term_no_linewrap();
+
+    if (clearscreen)
```

Comment:
@miczyg1 nit - this notation is error prone. Please change it to:

```
if (x) {
        do_something();
}
```

Reply:
@pietrushnic please look at the file history. I have just put it back as it was previously.